packages/beacon-state-transition/src/allForks/util/cachedBeaconState.ts

```diff
@@ -57,7 +60,11 @@ export function createCachedBeaconState<T extends allForks.BeaconState>(
   const cachedBalances = MutableVector.from(readonlyValues(state.balances));
   let cachedPreviousParticipation, cachedCurrentParticipation;
   const forkName = config.getForkName(state.slot);
-  if (forkName === ForkName.phase0) {
+  const isOnPhase0 = forkName === ForkName.phase0;
+  let currIndexedSyncCommittee: IndexedSyncCommittee | undefined;
+  let nextIndexedSyncCommittee: IndexedSyncCommittee | undefined;
+  const epochCtx = createEpochContext(config, state, cachedValidators, opts);
+  if (isOnPhase0) {
```

Comment:
Can we do here `forkName === ForkName.phase0` directly for consistency with other code?